lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -707,6 +728,29 @@ def write(chunk)
 
     private
 
+    def statusz_response
+      # TODO(davidbtucker): Add more status information here.
```

Comment:
Do we expect the amount of code for `statusz` would be significant? I wonder if it would be easier to maintain by splitting `statusz` related methods into a separate module/file like https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/master/lib/fluent/plugin/monitoring.rb

Reply:
Sure, I split out the function for creating the page into a separate module.